.github/PULL_REQUEST_TEMPLATE/accept-reject-pep.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Accept/Reject PEP
+about: Mark a draft Standards Track PEP as accepted or rejected
+title: PEP NNN: Mark as Accepted
```

Comment:
Viewing this file in the GitHub UI:

<img width="797" alt="image" src="https://user-images.githubusercontent.com/1324225/214009924-f23af56f-2d33-4bd7-94e2-4115392c3f10.png">

https://github.com/CAM-Gerlach/peps/blob/infra-pr-checklists/.github/PULL_REQUEST_TEMPLATE/accept-reject-pep.md

Where line 3 refers to the `title:` line. It doesn't like the repeated colon.

Editing in the UI and adding quotes, shows no warning in the preview:

```suggestion
title: "PEP NNN: Mark as Accepted"
```

Reply:
Oops, I was wondering about that and am usually pretty conservative quoting YAML, but in this case I forgot to check. Thanks!